### PR TITLE
Adding "File an EEO issue" form

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
Adding a File an EEO Issue form

## Additions

- Adding EEO form

## Testing

- http://localhost:7000/sub-pages/raise-an-eeo-issue/

## Review

- @sebworks 
- @anselmbradford

### CODE REVIEW

File: src/_includes/templates/forms/form-eeo-issue.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
+    <section class="block
+                    block__bg
+                    block__border
+                    block__flush-sides">
+        <form class="form-l js-validate-filters">
+            <div class="form-l_col form-l_col-1-2">
+                <label class="form-label-header" for="eeo_first-name">
```
Comment (anselmbradford) [inline src/_includes/templates/forms/form-eeo-issue.html]: If the ids are prefixed as `eeo_*`, should the `form` have an `eeo` class according to BEM?

File: src/_includes/templates/forms/form-eeo-issue.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+            </div>
+
+            <div class="form-l_col form-l_col-1-2">
+                <label class="form-label-header" for="eeo_fax">
+                    Fax Number
+                </label>
+                <input class="input__long"
+                       id="eeo_fax"
+                       type="text"
+                       required>
```
Comment (anselmbradford) [inline src/_includes/templates/forms/form-eeo-issue.html]: Fax not designated as required with `*`, but is `required` here.

File: src/_includes/templates/forms/form-eeo-issue.html [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+            <div class="form-l_col form-l_col-1">
+                <label for="eeo_complaint-desc">
+                    <span class="form-label-header">
+                        Nature of discrimination
+                    </span>
+                </label>
+                <textarea class="input__long"
+                          id="eeo_complaint-desc"
+                          required>
```
Comment (anselmbradford) [inline src/_includes/templates/forms/form-eeo-issue.html]: Nature of discrimination not designated as required with a `*`, but is `required` here.

File: src/_includes/templates/forms/form-eeo-issue.html [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+                </textarea>
+            </div>
+
+            <div class="form-actions form-actions__right-on-med ">
```
Comment (anselmbradford) [inline src/_includes/templates/forms/form-eeo-issue.html]: Add `form-l_col form-l_col-1` to classes to indent buttons so they are right-indent aligned with the form fields.

**Comment by anselmbradford** [comment]
According to http://cfpb.github.io/cf-grid/custom-demo/ we can nest a 6-column inside a 6-column and it won't have margins (see lower-left nesting example, which is two 6-columns inside a 4-column). I'm dealing with this in my form too, but the state and zip are indented when they shouldn't be:

screen shot 2015-08-05 at 10 11 19 am

Let me know if you find a solution that doesn't add new utility classes.

File: src/_includes/templates/forms/form-eeo-issue.html (hunk 1 above)
Comment (KimberlyMunoz) [inline src/_includes/templates/forms/form-eeo-issue.html]: I worry that `eeo` isn't descriptive enough to id a form. Would `eeo-form` be better and then I can change all the ids to `eeo-form_`?

Comment (anselmbradford) [inline src/_includes/templates/forms/form-eeo-issue.html]: Sounds good. This isn't current consistent across the forms, but I can update it to follow that format in mine.

**Comment by anselmbradford** [comment]
Realized the accessibility form mockup actually has the city, state, and zip as three equal-width columns. Do you know if the EEO one is the same?

screen shot 2015-08-05 at 2 32 59 pm

The fix is super easy to make it three equal-width columns:
1. Change https://github.com/cfpb/cfgov-refresh/pull/853/files#diff-4404eda9baa22a9df25829eaefa32754R47 to `form-l_col-1-3`.
2. Remove outer nesting div at https://github.com/cfpb/cfgov-refresh/pull/853/files#diff-4404eda9baa22a9df25829eaefa32754R58
3. Change to `form-l_col-1-3` [here](https://github.com/cfpb/cfgov-refresh/pull/853/files#diff-4404eda9baa22a9df25829eaefa32754R59) and [here](https://github.com/cfpb/cfgov-refresh/pull/853/files#diff-4404eda9baa22a9df25829eaefa32754R78).

If it needs to be laid out with the city field twice the width of the state and zip, it's almost as easy but requires some added CSS for a 1-4 column layout:
1. Add to `forms.less`:
```
.form-l_col-1-4 {
    .respond-to-min(1060px, {
        .grid_column(3);
    });
}
```
2. Remove outer nesting div, per step 2 above.
3. Set hierarchy as:
```
col-1-2 (city)
col-1-4 (state)
col-1-4 (zip)
```

**Comment by KimberlyMunoz** [comment]
Good catch, it is supposed to be a 1/3 column. I was going off an older wireframe for FOIA. Thanks. I'll get this and the suggestions above in a bit. I'm finishing up most of the sub-page browser tests for this section.

image

**Comment by KimberlyMunoz** [comment]
@anselmbradford: Issues fixed!

File: src/_includes/templates/forms/form-eeo-issue.html [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+                    Email Address*
+                </label>
+                <input class="input__long"
+                       id="eeo-form_email"
+                       type="email"
+                       required>
+            </div>
+
+            <div class="form-l_col form-l_col-1-2">
+                <label class="form-label-header" for="contact-time">
```
Comment (anselmbradford) [inline src/_includes/templates/forms/form-eeo-issue.html]: Should this be `eeo-form_contact-time`? Here and L:124

**Comment by anselmbradford** [comment]
One minor issue. Fix as you see fit, squash and merge :+1:

File: src/_includes/templates/forms/form-eeo-issue.html (hunk 5 above)
Comment (KimberlyMunoz) [inline src/_includes/templates/forms/form-eeo-issue.html]: Good catch!

**Pull request merged**